Automattic/newspack-newsletters
# feat: active-campaigns data report

### All Submissions:






### Changes proposed in this Pull Request:



Usage data report (see #1331) for ActiveCampaign. 

### How to test the changes in this Pull Request:

1. With ActiveCampaign set as the ESP, run `wp eval "echo print_r( (\Newspack_Newsletters::get_service_provider())->get_usage_report(1) , true );"` in the console – observe an appropriate array printed. If there's little recent data, use a higher argument to `get_usage_report` (the argument is the number of days-ago for the report).   

### Other information:
